packages/components/src/components/skeletons/utils.ts

```diff
@@ -20,8 +19,8 @@ export const shimmerEffect = css<{ elevation: Elevation }>`
     background: linear-gradient(
         90deg,
         ${({ theme, elevation }) => {
-            const start = theme[mapElevationToSkeletonBackground[elevation]];
-            const end = theme[mapElevationToSkeletonForeground[nextElevation[elevation]]];
+            const start = theme[mapElevationToSkeletonForeground[elevation]];
+            const end = theme[mapElevationToSkeletonBackground[elevation]];
```

Comment:
I've added comments to figma for corresponding color tokens, in case you belive linking the skeleton colors to backgrounds is a good idea. Otherwise just use those hex values for the light and dark themes.

Reply:
I think better would be to link it

Reply:
I think this is better, lets link them